Tests/WebSocketKitTests/WebSocketKitTests.swift

```diff
@@ -551,14 +528,126 @@ final class WebSocketKitTests: XCTestCase {
         try client.syncShutdown()
     }
 
-    var elg: EventLoopGroup!
+    
+    
+    // compression usage tests
```

Comment:
Let's remove this old (commented) testing code

Reply:
I just noticed the tests I added to websocket kit dont seem to be here ....!?

Reply:
Should be fixed now.